backend/service/github/github.go

```diff
@@ -273,3 +275,21 @@ func (s *svc) GetFile(ctx context.Context, ref *RemoteRef, path string) (*File,
 
 	return f, nil
 }
+
+func (s *svc) CompareCommits(ctx context.Context, ref *RemoteRef, compareSHA string) (scgithubv1.CommitCompareStatus, error) {
```

Comment:
we should nest the return `scgithubv1.CommitCompareStatus` in an object for future expansion.

can you add a message `CommitComparison` to the proto?